src/v2/guide/components-edge-cases.md

```diff
@@ -0,0 +1,311 @@
+---
+title: Handling Edge Cases
+type: guide
+order: 106
+---
+
+> This page assumes you've already read the [Components Basics](components.html). Read that first if you are new to components.
+
+<p class="tip">All the features on this page document the handling of edge cases, meaning unusual situations that sometimes require bending Vue's rules a little. Note however, that they all have disadvantages or situations where they could be dangerous. These are noticed in each case, so keep them in mind when deciding to use each feature.</p>
+
+## Element & Component Access
+
+In most cases, it's best to avoid reaching into other component instances or manually manipulating DOM elements. There are cases, however, when it can be appropriate.
+
+### Accessing the Root Instance
+
+In every subcomponent of a `new Vue` instance, this root instance can be accessed with the `$root` property. For example, in this root instance:
+
+```js
+// The root Vue instance
+new Vue({
+  data: {
+    foo: 1
+  },
+  computed: {
+    bar: function () { /* ... */ }
+  }
+  methods: {
+    baz: function () { /* ... */ }
+  }
+})
+```
+
+All subcomponents will now be able to access this instance and use it as a global store:
+
+```js
+// Get root data
+this.$root.foo
+
+// Set root data
+this.$root.foo = 2
+
+// Access root computed properties
+this.$root.bar
+
+// Call root methods
+this.$root.baz()
+```
+
+<p class="tip">This can be convenient for demos or very small apps with a handful of components. However, the pattern does not scale well to medium or large-scale applications, so we strongly recommend using <a href="https://github.com/vuejs/vuex">https://github.com/vuejs/vuex</a> to manage state in most cases.</p>
```

Comment:
I'd suggest this instead: `<a href="https://github.com/vuejs/vuex">vuex</a>`.

Reply:
Yeah, this was a mistake. 😅